packages/opentelemetry-node/test/spanprocessor.test.ts

```diff
@@ -462,6 +462,36 @@ describe('SentrySpanProcessor', () => {
       });
     });
 
+    it('adds transaction source `url` for HTTP_TARGET', async () => {
```

Comment:
l: Instead of adding these tests, maybe it is better to adjust all the tests starting here: https://github.com/getsentry/sentry-javascript/blob/af9d0f7677692feb2ca7eebd5a8c6f51a35f975a/packages/opentelemetry-node/test/spanprocessor.test.ts#L408 to simply also check the `source`? There are already tests for the different otel attribute variations, IMHO we could just add an `expect(sentrySpan?.transaction?.metadata.source).toBe('XXX');` check to all of them?

Reply:
I did this because source only applies for transactions - so I wanted an explicit scenario where we were checking the transaction (span.transaction when it's a transaction is a self reference) for this test. Gonna keep it like this for now, we can always update later on if need be.